### PR TITLE
Add `slow` marker to pytest

### PR DESCRIPTION
**Pull request by pentschev**
By adding the `slow` marker to `pytest` we can avoid using `--ignore-glob` on each test based on the condition of them being slow, providing also a clean output to users on skipped tests that may seem like they hang otherwise.

### CODE REVIEW

**Review by madsbk — APPROVED** [review]
LGTM, thanks @pentschev

**Comment by pentschev** [comment]
Thanks for review @madsbk , we'll still need an approval from @rapidsai/ops-codeowners .

**Review by raydouglass — APPROVED** [review]

**Pull request merged**
